dune/xt/common/test/main.hxx

```diff
@@ -78,6 +78,8 @@ int main(int argc, char** argv)
   MPIHelper::instance(argc, argv);
 #endif
 
+    DXTC_CONFIG = Configuration(argc, argv);
```

Comment:
I found this extremely helpful when playing with tests locally (this allows to pass a paramfile as comman line argument and use `DXT_CONFIG_GET` anywhere). Tests still seem to pass...
Any objections, @dune-community/dune-xt-common-devs?